modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -141,25 +139,27 @@ class Instrument_Manager extends \NDB_Menu_Filter
                     )
                 );
         }
-        $uploaded_file = $request->getUploadedFiles()['install_file'];
-        $filename      = $uploaded_file->getClientFilename();
-        $instrument    = pathinfo($filename)['filename'];
 
-        if ($this->instrumentExists($instrument)) {
-            return (new \Loris\Http\Response())
-                ->withStatus(409, 'Conflict')
+        // Do pre-processing before uploading files.
+        if (!$this->canWriteFiles()) {
```

Comment:
This is still not reachable because line 133 already returns with the exact same if condition.

Reply:
everything looks good from my end but i wanted to follow up on other comments. @driusan i don't see/understand this problem. can you confirm this is in fact resolved?

Reply:
@johnsaigle do you know anything about this?

Reply:
looks like it was resolved. The line used to be unreachable, because there was another if above it checking the same thing that the function call was checking.